### PR TITLE
feat: button for showing vaccination info

### PR DESCRIPTION
**Pull request by superskip**

### CODE REVIEW

**Review by Bekkalizer — APPROVED** [review]
LGTM! We need to adjust the fields shown in the modal, but seems like we can merge this now. We can test in the test environment wether the modal looks correct with the correct info in the vaccine list.

File: components/registration/vaccination-modal.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+</div>
+<div class="modal-body page">
+    <table class="dhis2-list-table-striped default-form-table">
+        <tr ng-repeat="vaccine in gridData">
+            <td>{{dateFromItem(vaccine)}}<br>
+                {{vaccine.recorder.name}}<div ng-if="!vaccine.performedByRecorder"><br>(Utført av en annen helseenhet)</div></td>
+            <td><div style="padding-bottom:5px">{{vaccine.atcCode}}({{vaccine.lotNumber}})
+        </tr>
+    </table>
+    <div ng-if="!gridData || gridData.length == 0">
```
Comment (helgeskjellevik) [inline components/registration/vaccination-modal.html:12]: FHI returns kanLevereUtData=false to mark that the organisation making the request is not authorised to view data on the provided pid. Id est; you can not make the assumption that no gridData = no vaccination without also checking on that boolean field.

File: scripts/external-lookup-services.js [hunk 2]
```diff
@@ -4863,6 +4863,33 @@ var externalLookupServices = angular.module('externalLookupServices', ['ngResour
...
+                }).then(function(response){
+                    return response.data;
+                },function(error) {
+                    var errorMsgHdr, errorMsgBody;
+                    errorMsgHdr = $translate.instant('error');
+
+                    errorMsgBody =  'Feil ved henting av vaksinedata:' + fNr;
+
+                    if(error.status == 403) {
+                        errorMsgBody = `Tjenesten Fiks prøvesvar er ikke tilgjengelig for deg.`;
```
Comment (helgeskjellevik) [inline scripts/external-lookup-services.js:4882]: Wrong service name.

Should also provide a better explanation text - something akin of the one for folkeregister search.
The end-user might get this if he/she lacks authorisation. Or alternatively if the organisation have failed to activate the service.

File: scripts/external-lookup-services.js [hunk 3]
```diff
@@ -4863,6 +4863,33 @@ var externalLookupServices = angular.module('externalLookupServices', ['ngResour
                 });
                 return promise;
             },
+            lookupVaccine: function(fNr, kommuneNr, userId) {
+                var url = '../' + DHIS2URL + '/vaksine/sok';
+                var promise = $http({
+                    method: 'POST',
+                    url: url,
+                    data: {fnr:fNr, kommunenr:kommuneNr, userid:userId},
+                    headers: {'Content-Type': 'application/json'}
```
Comment (helgeskjellevik) [inline scripts/external-lookup-services.js:4872]: Can you add a header 'ingress-csrf': '<value from cookie 'ingress-csrf'>

File: components/registration/registration-controller.js [hunk 4]
```diff
@@ -1463,7 +1463,11 @@ trackerCapture.controller('RegistrationController',
                             return DateUtils.getDateFromUTCString(item.vaccinationDate);
                         }
 
-                        $scope.cancel = function(){
+                        $scope.noVaccinesMessage = response.kanLeverUtData ?
+                            "Det er ingen registrerte vaksineringer på dette fødselsnummeret." :
+                            "Du har ikke de nødvendige rettighetene for å hente ut vaksineinformasjon.";
+
```
Comment (helgeskjellevik) [inline components/registration/registration-controller.js:1469]: Nitpick. 
Du har ikke de nødvendige rettighetene for å hente ut vaksineinformasjon på denne personen.

File: webpack.config.js [hunk 5]
```diff
@@ -16,8 +16,8 @@ try {
     console.warn('\nWARNING! Failed to load DHIS config:', e.message);
     console.info('Using default config');
     dhisConfig = {
-        baseUrl: 'http://localhost:8090/',
-        authorization: 'Basic ZG92cmU6VGVzdDEyMyE=' //c2VsOlRlc3QxMjMh sel:Test123!    //bWFya3VzOlN0b3BDb3ZpZDE5IQ== markus:StopCovid19!    // YWRtaW46ZGlzdHJpY3Q=' // admin:district
+        baseUrl: 'http://localhost:8080/',
+        authorization: 'Basic YWRtaW46ZGlzdHJpY3Q=' //c2VsOlRlc3QxMjMh sel:Test123!    //bWFya3VzOlN0b3BDb3ZpZDE5IQ== markus:StopCovid19!    // YWRtaW46ZGlzdHJpY3Q=' // admin:district
```
Comment (helgeskjellevik) [inline webpack.config.js:20]: Was this config change supposed to be part of the commit?

Comment (superskip) [reply webpack.config.js:20]: Ah, no :+1:

Comment (helgeskjellevik) [reply webpack.config.js]: What about the port change?
localhost:8080 > localhost:8090

Comment (superskip) [reply webpack.config.js]: Ahem, yes, that's also a debug artifact...

**Review by helgeskjellevik — APPROVED** [review]
lgtm

**Pull request merged**